build.sbt

```diff
@@ -446,7 +446,7 @@ lazy val akkaHttpBackend = (projectMatrix in file("akka-http-backend"))
   .settings(
     name := "akka-http-backend",
     libraryDependencies ++= Seq(
-      akkaHttp,
+      akkaHttp.cross(CrossVersion.for3Use2_13),
```

Comment:
there's no akka http for scala 3?

Reply:
unfortunately no as now https://github.com/akka/akka-http/pull/3901

Reply:
ah. Well if it works, that's the best we have. I've created https://github.com/softwaremill/sttp/issues/1241 to remove this in the future

Reply:
ah2. I see the cross-version doesn't seem to work, so we'll have to keep this PR open until there's a new release.

Reply:
Hi @adamw :)out of curiosity, what exactly is the problem here with akka-http for3Use2_13 workaround?

Reply:
There are some compilation errors with such setting, you can check CI run for more details
![image](https://user-images.githubusercontent.com/59844835/153036158-98696e4e-3e57-4fb3-bed8-d10d3a41668e.png)

Reply:
I see, it seems to be a problem of Akka dependency which is on provided scope in akka-http. akka-http expect _2.13 akka artefacts. You would have to probably add akka-stream also as for3Use2_13 🤔